src/containers/SubjectViewer.jsx

```diff
@@ -0,0 +1,252 @@
+    this.onMouseUp = this.onMouseUp.bind(this);
+    this.onMouseMove = this.onMouseMove.bind(this);
+    this.onMouseLeave = this.onMouseLeave.bind(this);
+    this.onWheel = this.onWheel.bind(this);
+    
+    //Other functions
+    this.getPointerXY = this.getPointerXY.bind(this);
+    this.getBoundingBox = this.getBoundingBox.bind(this);
+    
+    //Mouse or touch pointer
+    this.pointer = {
+      start: { x: 0, y: 0 },
+      now: { x: 0, y: 0 },
+      state: INPUT_STATE.IDLE,
+    };
+    
+    //Misc
+    this.tmpTransform = null;
+  }
+  
+  //----------------------------------------------------------------
+  
+  render() {
+    const transform = `scale(${this.props.scaling}) translate(${this.props.translationX}, ${this.props.translationY}) rotate(${this.props.rotation}) `;
+    
+    return (
+      <section className="subject-viewer" ref={(c)=>{this.section=c}}>
+        <svg
+          ref={(c)=>{this.svg=c}}
+          viewBox="0 0 100 100"
+          onMouseDown={this.onMouseDown}
+          onMouseUp={this.onMouseUp}
+          onMouseMove={this.onMouseMove}
+          onMouseLeave={this.onMouseLeave}
+          onWheel={this.onWheel}
+        >
+          <g transform={transform}>
+            <SVGImage src="https://panoptes-uploads.zooniverse.org/production/subject_location/97af440c-15d2-4fb1-bc18-167c9151050a.jpeg" />
+          </g>
+          <g className="developer-grid" transform={transform}>
+            {(()=>{
+              const MIN_VAL = -1000;
+              const MAX_VAL = 1000;
+              const STEP_VAL = 100;
+              const STYLE = { stroke: '#fff', strokeWidth: 2 };
+              const STYLE_ORIGIN = { stroke: '#c33', strokeWidth: 2 };
+              const STYLE_TEXT = { fill: '#c33', fontSize: '32px' }
+              const STYLE_TEXT_SHADOW = { fill: '#fff', fontSize: '32px' }
+              const arr = []
+              for (let v = MIN_VAL; v <= MAX_VAL; v += STEP_VAL) {
+                arr.push(<line x1={v} y1={MIN_VAL} x2={v} y2={MAX_VAL} style={STYLE} />);
+                arr.push(<line x1={MIN_VAL} y1={v} x2={MAX_VAL} y2={v} style={STYLE} />);
+              }
+              arr.push(<line x1={-STEP_VAL} y1={0} x2={STEP_VAL} y2={0} style={STYLE_ORIGIN} />);
+              arr.push(<line x1={0} y1={-STEP_VAL} x2={0} y2={STEP_VAL} style={STYLE_ORIGIN} />);
+              arr.push(<text x={2} y={0} style={STYLE_TEXT_SHADOW}>(0,0)</text>);
+              arr.push(<text x={-2} y={0} style={STYLE_TEXT_SHADOW}>(0,0)</text>);
+              arr.push(<text x={0} y={0} style={STYLE_TEXT}>(0,0)</text>);
+              return arr;
+            })()}
+          </g>
+        </svg>
+      </section>
+    );
+  }
+  
+  //----------------------------------------------------------------
+  
+  componentDidMount() {
+    //Make sure we monitor visible size of Subject Viewer.
+    window.addEventListener('resize', this.updateSize);
+    this.updateSize();
```

Comment:
I also added an event listener to `resize` in the new homepage PR to resize the background image. I doubt it will affect this component, but it's worth double checking.